diff --git a/node_modules/js-graph-algorithms/src/jsgraphs.js b/node_modules/js-graph-algorithms/src/jsgraphs.js
index 12fbde5..1b112a6 100755
--- a/node_modules/js-graph-algorithms/src/jsgraphs.js
+++ b/node_modules/js-graph-algorithms/src/jsgraphs.js
@@ -1202,7 +1202,4 @@ var jsgraphs = jsgraphs || {};
     jss.FordFulkerson = FordFulkerson;
 })(jsgraphs);

-var module = module || {};
-if(module) {
-	module.exports = jsgraphs;
-}
\ No newline at end of file
+module.exports = jsgraphs;